.idea/detekt.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="DetektPluginSettings">
+    <option name="configurationFiles">
+      <list>
+        <option value="$PROJECT_DIR$/config/detekt/detekt.yml" />
```

Comment:
One question, will this pickup the baseline file as well? Or maybe we should just remove it...

Reply:
It won't, but it can. It's a separate setting. 

However, I deliberately did not include it, as I thought it would be nice to keep being reminded of our "debt" in the IDE. If we add it, all those warnings will be ignored, and I'm afraid we might completely forget about them. Let me know if you disagree though!

Reply:
Well, my only concern is having different configuration in the terminal vs in the IDE... but it's probably fine yeah. In my mind, when we touch the files with those errors, we will probably modify them... Which I think is fine 👍

Reply:
I have been wondering if we need the baseline afterall, I think the ignores should be inline the code, it's easy to miss a warning is ignored if it's in the baseline file

Reply:
Wait, @vegaro's comment was not showing for me when I replied yesterday. So my reply is to @tonidero haha. Apologies. 
  
My thinking was that having the warnings in the baseline still show up in the IDE, is that it incentivizes us to fix them. 

However with 2 people on the fence about this approach, I'll add the baseline to the IntelliJ plugin settings. PR coming up.

Reply:
For reference: https://github.com/RevenueCat/purchases-android/pull/1798